app/build.gradle.kts

```diff
@@ -25,6 +25,10 @@ plugins {
     id("com.google.android.gms.oss-licenses-plugin")
     alias(libs.plugins.baselineprofile)
     alias(libs.plugins.roborazzi)
+    // Serialization is used for type-safe navigation.
```

Comment:
I don't think this is true, it works fine since you are in a regular Gradle module:

```kotlin
plugins {
 alias(libs.plugins.kotlin.serialization)
}
```

```toml
[plugins]
kotlin-jvm = { id = "org.jetbrains.kotlin.jvm", version.ref = "kotlin" }
```

Reply:
Oh nice, thanks for spotting that. Fixed.